src/object/srv_ec.c

```diff
@@ -111,9 +111,11 @@ obj_ec_metrics_process(daos_iod_t *iod, struct obj_io_desc *oiod, struct daos_oc
  * split it for different targets before dispatch.
  */
 int
-obj_ec_rw_req_split(daos_unit_oid_t oid, uint64_t dkey_hash, struct obj_iod_array *iod_array,
-		    uint32_t iod_nr, uint32_t start_shard, uint32_t max_shard, uint32_t leader_id,
-		    void *tgt_map, uint32_t map_size, struct daos_oclass_attr *oca, uint32_t tgt_nr,
+obj_ec_rw_req_split(daos_unit_oid_t oid, uint32_t start_tgt,
+		    struct obj_iod_array *iod_array,
+		    uint32_t iod_nr, uint32_t start_shard, uint32_t max_shard,
```

Comment:
what's the different between two arg "start_tgt" and "start_shard"? seems be the same? they looks confuse, can just use one?

Reply:
start_shard is from RPC, which is the 1st shard from the RPC, I assume this is the parity start_tgt is the start shard of the ec group(0th data shard). Hmm, they are probably the same in most common case.   Though the patch does not change the original logic. 

I will check this in the following patch.

Reply:
"I assume this is the parity" this server in processing is the leader shard, but possibly not parity shard (client possibly select data shard as leader for update).
And looks you added "start_tgt" is completely same as start_shard.